.rubocop.yml

```diff
@@ -0,0 +1,59 @@
+AllCops:
+  Exclude:
+    - "README.md"
+    - "Guardfile"
+    - "Rakefile"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 40
```

Comment:
[Optional]
You should not make a change to the config file. As the previous reviewer did not mention that I am marking it as optional. But remember, you should not make any change to this file.